scripts/build.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+function cleanup {
+  rm -rf build
+  rm AppleDevIDApp.p12
+  rm notarize_config.json
+  security delete-keychain percy.keychain
+}
+
+npm install -g pkg
+
+yarn build
+
+yarn install
+
+# Remove type from package.json files
+sed -i '' '/"type": "module",/d' ./package.json
+cd packages && sed -i '' '/"type": "module",/d' ./*/package.json && cd ..
+
+echo "import { cli } from '@percy/cli';\
+$(cat ./packages/cli/dist/percy.js)" > ./packages/cli/dist/percy.js
+
```

Comment:
can we move L16-L21 also to code?

Reply:
these lines can’t be moved behind env

Reply:
[optional] ideally we can using build script ( `yarn build` ) we can use the same `env` to differentiate.

Give it a try, otherwise we can do in followup PR

Reply:
it might take time let do in followup PR